.github/workflows/build.yml

```diff
@@ -296,6 +302,25 @@ jobs:
           cp -R 'tmp.package/Assets/TextMesh Pro' Assets/
           rm -rf tmp.plugin tmp.package
 
+
+      - name: Manually delete Zapbox package
```

Comment:
This is going to be problematic for people not using CI. At the absolute least, it needs to go into the README!

Reply:
Good point! this step should be very temporary, I can park this PR until we get a newer SDK version from Zappar.

Reply:
I'm ok with pushing it, but I'd recommend a message on Discord as well (pinned).

Reply:
CC @tangobravo this is our only blocker for merging initial support and getting you some xcode builds :)